inst/rmarkdown/templates/trb/resources/template.tex

```diff
@@ -0,0 +1,80 @@
+% Repository:  https://github.com/chiehrosswang/TRB_LaTeX_tex
+%
+% Transportation Research Board conference paper template
+% version 4.0 Lite (updates made to be compatible in Overleaf and ShareLaTeX)
+%
+%
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbarticle}
+\usepackage{graphicx}
+\usepackage{booktabs}
+
+\newread\somefile
+\usepackage{xparse}
+
+\usepackage{natbib}
+\bibliographystyle{unsrtnat}
+\setcitestyle{round}
+
+% \usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
+% For TRB version hide links
+\usepackage[hidelinks]{hyperref}
+
+% Put here what will go to headers as author
+\AuthorHeaders{$runningheader$}
+\title{$title$}
+
+% TODO: add macros for easier formatting of \author.
+\author{%
+  $for(author)$
+  \textbf{$author.name$}\\
+  \textit{$if(author.note)$$author.note$\\$endif$}
+  $author.position$\\
+  $author.affiliation$\\
+  $author.email$\\
+  \hfill\break
+  $endfor$
+}
+
+% If necessary modify the number of words per table or figure default is set to
+% 250 words per table
+% \WordsPerTable{250}
+
+% If words are counted manually, put that number here. This does not include
+% figures and tables. This can also be used to avoid problems with texcount
+% program i.e. if one does not have it installed.
+\TotalWords{$wordcount$}
+
```

Comment:
Here before begin document, we need to add some Pandoc required content. 

* For highlighting of code chunk if there is some. Try `echo = TRUE` on your document, and you'll have an error. 
```
$if(highlighting-macros)$
$highlighting-macros$
$endif$
```

* If anyone want to use `header-includes` feature of Pandoc, or `pdf_document`, `includes:` argument for `in_header`, `after_body` and `before_body` insertion. See other template in the repo.  Sometimes it is missing, but it can be useful to have them all already in the template. You already added one below. There is two more
  * https://github.com/rstudio/rticles/blob/4605713e221f1a8c16b3c5686716cc53c1fa78c4/inst/rmarkdown/templates/agu/resources/template.tex#L97-L99
  * https://github.com/rstudio/rticles/blob/f14a79bed9597c7ea559a20ae47c8facd20066c3/inst/rmarkdown/templates/arxiv/resources/template.tex#L94-L96

* Some stuff are needed to support Pandoc citeproc. 
https://github.com/rstudio/rticles/blob/f14a79bed9597c7ea559a20ae47c8facd20066c3/inst/rmarkdown/templates/arxiv/resources/template.tex#L55 
Are we supporting it ? 

* If list are to be used, this is required 
https://github.com/rstudio/rticles/blob/4aa0a44e4f41b02cfb1d5c1cdc80043b5f42e502/inst/rmarkdown/templates/rjournal/resources/RJwrapper.tex#L8-L9
By list I mean
```markdown
* Try
* A list
* like this
```


There may be more but it depends on which feature from Pandoc we want to support. The default template clearly have some more stuff : https://github.com/jgm/pandoc/blob/master/data/templates/default.latex

Reply:
Got it. What is the advantage to supporting citeproc?

Reply:
Just that it can work with Pandoc citation processing and CSL. But I understand one should use `natbib` for this template right ?

Reply:
I added what was missing